src/com/goide/dlv/DlvStackFrame.java

```diff
@@ -148,20 +147,65 @@ public void computePresentation(@NotNull XValueNode node, @NotNull XValuePlace p
           node.setPresentation(icon, presentation, false);
           return;
         }
+
+        if (variable.type.startsWith("struct struct")) {
+          String type = variable.type.replace("struct struct", "struct").trim();
+          String value = variable.value.replace(type,  "").trim();
+          node.setPresentation(icon, type, value, false);
+          return;
+        }
+
+        if (variable.type.startsWith("struct []")) {
+          String type = variable.type.replace("struct []", "[]").trim();
+          String value = variable.value.replaceAll("(.*\\scap:\\s\\d+,\\s)",  "").trim();
+          node.setPresentation(icon, type, value, false);
+          return;
+        }
+
         node.setPresentation(icon, variable.type, variable.value, false);
       }
 
       @Nullable
       private XValuePresentation getPresentation() {
-        String type = variable.type;
+        final String type = variable.type;
         final String value = variable.value;
-        if (NUMBERS.contains(type)) return new XNumericValuePresentation(value);
-        if ("struct string".equals(type)) return new XStringValuePresentation(value);
+        if (NUMBERS.contains(type)) return new XValuePresentation() {
```

Comment:
For the type of the value. Right now one can't distinguish between int, uint, int64 and so on, this should help out, see the screenshot.